src/services/authenticator.js

```diff
@@ -51,29 +50,17 @@ function Authenticator() {
     || 'Invalid token. Please enter your authentication token.';
 
   this.logout = async (opts = {}) => {
-    const path = `${os.homedir()}/.forestrc`;
-
-    return new P((resolve, reject) => {
-      fs.stat(path, (err) => {
-        if (err === null) {
-          fs.unlinkSync(path);
-
-          if (opts.log) {
-            console.log(chalk.green('👍  You\'re now unlogged 👍 '));
-          }
-
-          resolve();
-        } else if (err.code === 'ENOENT') {
-          if (opts.log) {
-            logger.error('You are not logged');
-          }
-
-          resolve();
-        } else {
-          reject(err);
-        }
-      });
-    });
+    const forestrcPath = `${os.homedir()}/.forestrc`;
+    const isForestLoggedIn = this.getVerifiedToken(forestrcPath);
+    const lumbercPath = `${os.homedir()}/.lumberrc`;
+    const isLumberLoggedIn = this.getVerifiedToken(lumbercPath);
+
```

Comment:
[Readability] I prefer to start variable names by what kind of object they represent and then add adjectives.
```suggestion
    const pathForestrc = `${os.homedir()}/.forestrc`;
    const pathLumberc = `${os.homedir()}/.lumberrc`;
    const isLoggedInWithForestCLI = this.getVerifiedToken(forestrcPath);
    const isLoggedInWithLumberCLI = this.getVerifiedToken(lumbercPath);

```